test/bats/cli-test.bats

```diff
@@ -47,3 +47,13 @@ load helpers
     run bin/ratify verify -c $RATIFY_DIR/config.json -s wabbitnetworks.azurecr.io/test/all-in-one-image:signed
     assert_cmd_verify_success
 }
+
+@test "dynamic plugin verifier test" {
```

Comment:
should we duplicate this test for stores to add the same coverage for feature flag enabled/disabled.

Reply:
Good catch, thanks! Added in 675989c98b8052152d4dc6715c6e92804ae5d64f

It's passing in e2e tests, but all the new e2e aks tests are now failing on my PR due to permissions